src/blog/2020/001-dmn-js-8.md

```diff
@@ -0,0 +1,92 @@
+---
+
+title: DMN 1.3 support in dmn-js
+layout: blogpost.hbs
+
+slug: 2020-dmn-js-8-0-0
+author: Philipp Fromme<https://github.com/philippfromme>
+
+published: 2020-02-28 12:00
+
+releases:
+  - 'dmn-js@8.0.0'
+
+---
+
+
+<p class="introduction">
+  We're excited to announce the latest release of our DMN modeling toolkit, which supports the latest DMN specification.
+</p>
+
+<!-- continue -->
+
+On Tuesday, February 25th, the Object Management Group (OMG) officially released the latest version of the Decision Model and Notation (DMN) specification. DMN 1.3 adds Diagram Interchange (DMN DI) for specifying the layout of the shapes and edge of a DMN diagram. Previous versions of dmn-js already allow the specification of the layout of DMN diagrams (Decision Requirement Diagrams), made possible by an extension that is not part of the DMN standard. With the release of DMN 1.3 and our latest dmn-js release, this extension is no longer necessary. To illustrate this, let's compare a DMN 1.1 XML to a DMN 1.3 XML:

```

Comment:
> February 25th

that's pending, right?

Reply:
> shapes and edge

shapes and **edges**

Reply:
> the layout of DMN diagrams (Decision Requirement Diagrams)

do you mean DRD diagrams (because you explain the abbreviation in brackets)?

Reply:
> With the release of DMN 1.3 and our latest dmn-js release

doubling of _release_, maybe use _version_ one time?

Reply:
> > February 25th
> 
> that's pending, right?

This is a placeholder.

Reply:
If we release before the actual release of DMN 1.3 this will be adjusted.